src/Components/UI/Calculator.js

```diff
@@ -1,47 +1,110 @@
-/* eslint-disable react/prefer-stateless-function */
+/* eslint-disable react/prefer-stateless-function, react/destructuring-assignment, no-nested-ternary, max-len */
+// /* eslint-disable */
```

Comment:
- [ ] Kindly remove the disable linters as linters help us write clean industry acceptable code.